app/routes/events/components/EventAdministrate/Abacard.tsx

```diff
@@ -1,56 +1,36 @@
-import { usePreparedEffect } from '@webkom/react-prepare';
-import { get } from 'lodash';
-import qs from 'qs';
-import { useLocation, useParams } from 'react-router-dom';
+import { useParams } from 'react-router-dom';
 import { markUsernamePresent } from 'app/actions/EventActions';
-import { autocomplete } from 'app/actions/SearchActions';
 import Validator from 'app/components/UserValidator';
-import { getRegistrationGroups, selectEventById } from 'app/reducers/events';
+import { selectEventById, selectRegistrationGroups } from 'app/reducers/events';
 import { useAppDispatch, useAppSelector } from 'app/store/hooks';
 import styles from './Abacard.css';
+import type { AdministrateEvent } from 'app/store/models/Event';
 
 const Abacard = () => {
-  const { eventId } = useParams<{ eventId: string }>();
-  const event = useAppSelector((state) => selectEventById(state, { eventId }));
+  const { eventId } = useParams<{ eventId: string }>() as { eventId: string };
+  const event = useAppSelector((state) =>
+    selectEventById<AdministrateEvent>(state, eventId),
+  );
   const { registered } = useAppSelector((state) =>
-    getRegistrationGroups(state, {
+    selectRegistrationGroups(state, {
       eventId,
     }),
   );
 
-  const location = useLocation();
-  const query = qs.parse(location.search, {
-    ignoreQueryPrefix: true,
-  });
-
   const dispatch = useAppDispatch();
 
-  usePreparedEffect(
-    'fetchEventAbacard',
-    () => {
-      if (query && typeof query === 'string') {
-        return dispatch(autocomplete(query, ['users.user']));
-      }
-    },
-    [query],
-  );
```

Comment:
I don't know what this was supposed to do. `typeof query` is never a string

Reply:
you never know with this webapp! 😆